provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -1507,6 +1507,9 @@ spec:
             {% if config.net_config.disable_wait_for_network %}
             - name: DISABLE_WAIT_FOR_NETWORK
               value: 'True'
+            {% else %}
+            - name: DURATION_WAIT_FOR_NETWORK
+              value: {{'\"' + config.net_config.duration_wait_for_network + '\"'}}
```

Comment:
Why do we need to put this in \"\"? The DURATION_WAIT_FOR_NETWORK env var will take the integer value of config.net_config.duration_wait_for_network and which will automatically be available to the launch-hostagent script as a string (and which we are taking care of converting to an integer in the script).

Reply:
Change config.net_config.duration_wait_for_network to: config.net_config.duration_wait_for_network|json